lang_tests/run.rs

```diff
@@ -0,0 +1,42 @@
+use std::{env, path::PathBuf, process::Command};
+
+use lang_tester::LangTester;
+use lazy_static::lazy_static;
+use regex::{Regex, RegexBuilder};
+
+lazy_static! {
+    static ref EXPECTED: Regex = RegexBuilder::new("^\\[(.*?)^\\][ \t]*$")
+        .multi_line(true)
+        .dot_matches_new_line(true)
+        .build()
+        .unwrap();
+}
+
+fn main() {
+    LangTester::new()
+        .test_dir("lang_tests")
+        .test_file_filter(|p| {
+            p.extension().unwrap().to_str().unwrap() == "bf"
+        })
+        .test_extract(|s| {
+            EXPECTED
+                .captures(s)
+                .map(|x| x.get(1).unwrap().as_str().trim().to_owned())
+        })
+        .test_cmds(|p| {
+            // We call target/[debug|release]/yksom directly, because it's noticeably faster than
+            // calling `cargo run`.
+            let mut yksom_bin = PathBuf::new();
+            yksom_bin.push(env::var("CARGO_MANIFEST_DIR").unwrap());
+            yksom_bin.push("target");
```

Comment:
Can we use `CARGO_BIN_EXE_ykbf` here? I've never used it, but sounds like it could make this code a lot shorter:

https://doc.rust-lang.org/cargo/reference/environment-variables.html

Reply:
I did not know about that variable! Fixed in https://github.com/softdevteam/ykbf/pull/4/commits/936266e6308294e7a3b3c1f6a4a7e72e562cdc2a.